presentation/toc/english.html

```diff
@@ -87,6 +87,12 @@ <h1 id="rust-in-three-days">Rust in Three Days</h1>
 <ul>
 <li><a href="../index.html?chapter=community-map&amp;locale=en-US">Community-Overview</a></li>
 </ul>
+<h1 id="Rust & WebAssembly">Rust & WebAssembly</h1>
+<ul>
+  <li><a href="../index.html?chapter=wasm_01_introduction&amp;locale=en-US">Introduction</a></li>
+  <li><a href="../index.html?chapter=wasm_02_setup&amp;locale=en-US">Setup</a></li>
+  <li><a href="../index.html?chapter=wasm_03_game_of_life&amp;locale=en-US">Game of Life</a></li>
+</ul>
```

Comment:
These changes should be done by editing `english.md` and then building it with `pandoc` (or a similar markdown compiler).